### DISCUSSION TITLE
Please add unique login signature to admin tool signin

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#4228] [Q&A]
Unfortunately I find myself in a situation where I can never simultaneously remember my PB Admin password as well as my normal site password.  Firefox is overwriting the other whenever one of them changes.  I'll try to find time this weekend for research a method for doing that.  I estimate that most of the work there is just in finding a good modern method.

Now that I think about it, I could probably implement that into my main product signin.  But it seems like something that many people will bump into.

**Comment by ganigeorgiev** [comment]
I've actually remember attempting to fix this in the past in relation to https://github.com/pocketbase/pocketbase/discussions/2562 but I wasn't able to find a reliable solution without hacky workarounds like using custom content types or non-standard input elements with contenteditable.
If you manage to find a fix, feel free to create a PR and I'll try to review it.
